docs/content/development/adding-post-breach-actions.md

```diff
@@ -39,9 +39,9 @@ class MyNewPba(PBA):
 
 #### Implementation
 
-If your PBA consists only of simple shell commands, you can reuse the generic PBA by passing the commands into the constructor. See the `add_user.py` PBA for reference.
+If your PBA consists only of simple shell commands, you can reuse the generic PBA by passing the commands into the constructor. See the `timestomping.py` PBA for reference.
```

Comment:
Timestomping uses scripts which is slightly more complicated.

Job scheduling and account discovery, (1) exist for both Windows and Linux, and (2) are the easiest ones. Let's use one of those as an example here. You choose. :)